src/styles/blocks/card.css

```diff
@@ -0,0 +1,119 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+
+  max-width: 200px;
+  padding: 0 16px 16px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 32px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: var(--main-text-color);
+}
+
+.card__title {
+  margin: 40px 0 0;
+}
+
+.card__product-code {
+  margin: 4px 0 0;
+}
+
+.product-code {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+
+  color: var(--secondary-text-color);
+}
+
+.card__paragraph {
+  margin: 0;
+}
+
+.card__rate-block {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-top: 16px;
+
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__rate-reviews {
+  margin: 0;
+}
+
+.card__price-block {
+  display: flex;
+  justify-content: space-between;
+
+  margin: 24px 0 0;
+}
+
+.card__price {
+  margin: 0;
+
+  font-size: 12px;
+  line-height: 18px;
+
+  color: var(--secondary-text-color);
+}
+
+.card__price-sum {
+  margin: 0;
+
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+
+  color: var(--main-text-color);
+}
+
+.card__buy-button {
+  cursor: pointer;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  margin: 16px 0 0;
+  height: 40px;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-decoration: none;
+  color: #fff;
+
+  background-color: var(--blue-accent-color);
+  border: 1px solid var(--blue-accent-color);
+  border-radius: 5px;
+}
+
+.card__buy-button:hover {
+  transition: all 0.5s ease-out;
```

Comment:
don't add transition only to hover rule, it will not work proper way, move it to `.card__buy-button`